Fix build on 32-bit architectures with Mesa 18.3.  Patch taken from upstream:

https://git.enlightenment.org/core/efl.git/commit/?id=0d2b624f1e24240a1c4e651aa1cfe9a8dd10a573

diff --git a/src/lib/evas/Evas_GL.h b/src/lib/evas/Evas_GL.h
index b6b642400f..4f67b1695f 100644
--- a/src/lib/evas/Evas_GL.h
+++ b/src/lib/evas/Evas_GL.h
@@ -4272,9 +4272,11 @@ typedef signed int       GLfixed;      // Changed khronos_int32_t
 
 #ifndef GL_ES_VERSION_2_0
 /* GL types for handling large vertex buffer objects */
-#include <stddef.h>
+# ifndef GL_VERSION_1_5
+#  include <stddef.h>
 typedef ptrdiff_t GLintptr;     // Changed khronos_intptr_t
 typedef ptrdiff_t GLsizeiptr;   // Changed khronos_ssize_t
+# endif
 #endif
 
 /* Some definitions from GLES 3.0.
